lib/Db/QueueContentItemMapper.php

```diff
@@ -50,4 +50,21 @@ public function removeFromQueue(QueueContentItem $item): void {
 			->where($qb->expr()->eq('id', $qb->createPositionalParameter($item->getId())))
 			->executeStatement();
 	}
+
+	/**
+	 * @throws \OCP\DB\Exception
+	 * @return array<string, int>
+	 */
+	public function count() : array {
+		$qb = $this->db->getQueryBuilder();
+		$result = $qb->select($qb->func()->count('id', 'count'), 'app_id', 'provider_id')
+			->from($this->getTableName())
+			->groupBy('app_id', 'provider_id')
+			->executeQuery();
```

Comment:
db exception should be caught here since this would be called by a user facing function

Reply:
this needs to be caught still